src/analyzer/plugins/analyzerqueenmarybeats.cpp

```diff
@@ -39,9 +39,7 @@ DFConfig makeDetectionFunctionConfig(int stepSize, int windowSize) {
 } // namespace
 
 AnalyzerQueenMaryBeats::AnalyzerQueenMaryBeats()
-        : m_iSampleRate(0),
-          m_windowSize(0),
-          m_stepSize(0) {
+        : m_iSampleRate(0) {
```

Comment:
Sure, actually I removed this by mistake because of my lack of git skills.
I had first branched out you beatmap_48 breanch since that's wasnt on master yet and I wanted to use it, then I tried to revert that after merging master again and this went through somehow.